osquery/tables/system/windows/registry.cpp

```diff
@@ -227,9 +227,11 @@ inline void explodeRegistryPath(const std::string& path,
                                 std::string& rHive,
                                 std::string& rKey) {
   auto toks = osquery::split(path, kRegSep);
-  rHive = toks.front();
-  toks.erase(toks.begin());
-  rKey = osquery::join(toks, kRegSep);
+  if (toks.size()) {
```

Comment:
nitpick, do you mind updating this to use `if (!toks.empty()) {`

Reply:
Done.